GUI/Types/Renderer/GLSceneViewer.cs

```diff
@@ -530,8 +530,14 @@ private void OnRenderModeDrawItem(object sender, DrawItemEventArgs e)
 
             if (mode.IsHeader)
             {
-                e.Graphics.FillRectangle(SystemBrushes.Window, e.Bounds);
-                e.Graphics.DrawString(mode.Name, renderModeBoldFont, SystemBrushes.ControlText, e.Bounds);
+                var headerBrush = new SolidBrush(MainForm.DarkModeCS.OScolors.Surface);
+                var textBrush = new SolidBrush(MainForm.DarkModeCS.OScolors.TextActive);
+
+                e.Graphics.FillRectangle(headerBrush, e.Bounds);
+                e.Graphics.DrawString(mode.Name, renderModeBoldFont, textBrush, e.Bounds);
+
+                headerBrush.Dispose();
+                textBrush.Dispose();
```

Comment:
using var?

Reply:
Yes. and using var doesnt require `() {}` like below so you can clean that up too.